examples/postgres/getting_started_step_3/src/lib.rs

```diff
@@ -15,16 +15,14 @@ pub fn establish_connection() -> PgConnection {
     dotenv().ok();
 
     let database_url = env::var("DATABASE_URL").expect("DATABASE_URL must be set");
-    PgConnection::establish(&database_url).expect(&format!("Error connecting to {}", database_url))
+    PgConnection::establish(&database_url)
+        .unwrap_or_else(|_| panic!("Error connecting to {}", database_url))
 }
 
 pub fn create_post(conn: &PgConnection, title: &str, body: &str) -> Post {
     use schema::posts;
 
-    let new_post = NewPost {
-        title: title,
-        body: body,
-    };
+    let new_post = NewPost { title, body };
```

Comment:
I've just been allowing this elsewhere to avoid the git churn, but I guess it's not too bad if it's only in two places.